packages/forma-36-codemod/transforms/v4-button.js

```diff
@@ -0,0 +1,128 @@
+const {
+  getComponentLocalName,
+  renameProperties,
+  hasProperty,
+  addProperty,
+  changeProperties,
+  updatePropertyValue,
+  updateIcons,
+  addIconImports,
+  changeImport,
+  deleteProperty,
+  updateTernaryValues,
+  warningMessage,
+} = require('../utils');
+const { getFormaImport, shouldSkipUpdateImport } = require('../utils/config');
+
+module.exports = function (file, api) {
+  const j = api.jscodeshift;
+
+  let source = file.source;
+
+  const componentName = getComponentLocalName(j, source, {
+    componentName: 'Button',
+    importName: getFormaImport(),
+  });
+
+  if (!componentName) {
+    return source;
+  }
+
+  const usedIcons = [];
+
+  source = changeProperties(j, source, {
+    componentName,
+    fn(attributes) {
+      let modifiedAttributes = attributes;
+
+      modifiedAttributes = renameProperties(modifiedAttributes, {
+        renameMap: {
+          buttonType: 'variant',
+          disabled: 'isDisabled',
+          loading: 'isLoading',
+        },
+      });
+
+      modifiedAttributes = updatePropertyValue(modifiedAttributes, {
+        j,
+        propertyName: 'variant',
+        propertyValue: (value) => {
+          const valueMap = {
+            [undefined]: 'primary',
+            muted: 'secondary',
+            warning: 'secondary',
+            naked: 'transparent',
+          };
+          if (value.value !== undefined && valueMap[value.value]) {
+            if (value.value === 'warning') {
+              warningMessage(
+                'warning variant was deprecated, we updated the value to secondary please change this value if needed',
```

Comment:
Could we add some styling to this warning message to make it more readable, e.g.:
"Warning" button variant was deprecated. We updated the value to "Secondary". Please check your implementation.